app/cdap/components/WrangleHome/Components/OngoingDataExplorations/index.tsx

```diff
@@ -0,0 +1,189 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core/';
+import MyDataPrepApi from 'api/dataprep';
+import { getCategorizedConnections } from 'components/Connections/Browser/SidePanel/apiHelpers';
+import {
+  IConnectionWithConnectorType,
+  IExistingExplorationCard,
+  IConnectionsList,
+  IWorkspace,
+} from 'components/WrangleHome/Components/OngoingDataExplorations/types';
+import { getUpdatedExplorationCards } from 'components/WrangleHome/Components/OngoingDataExplorations/utils';
+import OngoingDataExplorationsCard from 'components/WrangleHome/Components/OngoingDataExplorationsCard';
+import T from 'i18n-react';
+import React, { useCallback, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { forkJoin } from 'rxjs/observable/forkJoin';
+import { of } from 'rxjs/observable/of';
+import { defaultIfEmpty, switchMap } from 'rxjs/operators';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { IExplorationCardDetails } from '../OngoingDataExplorationsCard/types';
+
+export default function() {
+  const [onGoingExplorationsData, setOnGoingExplorationsData] = useState<IExplorationCardDetails[]>(
+    []
+  );
+
+  const getOngoingData = useCallback(async () => {
+    const connectionsWithConnectorTypes: Map<
+      string,
+      IConnectionsList[]
+    > = await getCategorizedConnections();
+    const connectionsWithConnectorTypeData: IConnectionWithConnectorType[] = [];
+    for (const connectorName of connectionsWithConnectorTypes.keys()) {
+      const values = connectionsWithConnectorTypes.get(connectorName);
+      const connections = values.map((eachValue) => ({
+        name: eachValue.name,
+        connectorType: eachValue.connectionType,
+      }));
+      connectionsWithConnectorTypeData.push(...connections);
+    }
+
+    const findConnectorType = (connection: string) => {
+      if (connection) {
+        const matchedConnection: IConnectionWithConnectorType = connectionsWithConnectorTypeData.find(
+          (eachConnection) => eachConnection.name === connection.replace('_', ' ')
+        );
+        return matchedConnection ? matchedConnection.connectorType : undefined;
+      }
+      return T.translate(
+        'features.WranglerNewUI.OnGoingDataExplorations.labels.importedDataset'
+      ).toString();
+    };
+
+    const explorationData: IExistingExplorationCard[] = [];
+
+    // Getting the workspace name, path ,workspaceId and name from MyDataPrepApi.getWorkspaceList API and
+    //  using these in params and requestBody to get Data quality from MyDataPrepApi.execute API
+
+    MyDataPrepApi.getWorkspaceList({
+      context: 'default',
```

Comment:
This should be the current namespace, not `'default'`.

Reply:
Done